components/dial/stories/dial.stories.js

```diff
@@ -28,7 +28,15 @@ export default {
 			},
 			control: "text",
 		},
-		isFocused,
+		isFocusVisible: {
+			name: "Focused",
+			type: { name: "boolean" },
+			table: {
+				type: { summary: "boolean" },
+				category: "State",
+			},
+			control: "boolean",
+		},
```

Comment:
What would you think of doing this to extend the isFocused definition but still allow the FocusVisible variable name:

```suggestion
		isFocusVisible: isFocused,
```

Reply:
👏 ooooo that's fantastic! Yes! https://github.com/adobe/spectrum-css/pull/2833/commits/92eb2de7ff23cc4876803b6598561a2bd8b2d1d2